### PR TITLE
dev-qt/qtvirtualkeyboard: New Package

### PR DESCRIPTION
**Pull request by a17r**
Follow-up to https://github.com/gentoo/qt/pull/129 due to timeout

Rebase live versions, 5.7.1 version bump
Add missing DEPEND, fix USE=-handwriting
Disable LINGUAS that force-enable 3rdparty libs

KDE has made this a runtime requirement for Plasma 5.10 (5.10.1 release scheduled for tomorrow).

### CODE REVIEW

File: dev-qt/qtvirtualkeyboard/qtvirtualkeyboard-5.7.1.ebuild [hunk 1]
```diff
@@ -14,27 +14,27 @@ if [[ ${QT5_BUILD_TYPE} == release ]]; then
...
 	local myqmakeargs=(
-		$(usex spell '' 'CONFIG+=disable-hunspell')
-		$(usex xcb '' 'CONFIG+=disable-desktop')
-		$(usex handwriting 'CONFIG+=lipi-toolkit')
+		$(usex handwriting "CONFIG+=lipi-toolkit" "")
+		$(usex spell "" "CONFIG+=disable-hunspell")
+		$(usex xcb "" "CONFIG+=disable-desktop")
 	)
 
 	for x in ${PLOCALES}; do
```
Comment (Pesa) [inline dev-qt/qtvirtualkeyboard/qtvirtualkeyboard-5.7.1.ebuild]: use `l10n_get_locales`

Comment (Pesa) [inline dev-qt/qtvirtualkeyboard/qtvirtualkeyboard-5.7.1.ebuild]: use `l10n_get_locales`, and `local` for the variable

File: dev-qt/qtvirtualkeyboard/qtvirtualkeyboard-5.7.1.ebuild [hunk 2]
```diff
@@ -14,27 +14,27 @@ if [[ ${QT5_BUILD_TYPE} == release ]]; then
...
 
 src_configure() {
-
 	local myqmakeargs=(
-		$(usex spell '' 'CONFIG+=disable-hunspell')
-		$(usex xcb '' 'CONFIG+=disable-desktop')
-		$(usex handwriting 'CONFIG+=lipi-toolkit')
+		$(usex handwriting "CONFIG+=lipi-toolkit" "")
+		$(usex spell "" "CONFIG+=disable-hunspell")
+		$(usex xcb "" "CONFIG+=disable-desktop")
```
Comment (Pesa) [inline dev-qt/qtvirtualkeyboard/qtvirtualkeyboard-5.7.1.ebuild]: don't quote if unneeded (`CONFIG+=...`)

File: dev-qt/qtvirtualkeyboard/qtvirtualkeyboard-5.7.1.ebuild [hunk 3]
```diff
@@ -14,27 +14,27 @@ if [[ ${QT5_BUILD_TYPE} == release ]]; then
...
-IUSE="+spell +xcb handwriting"
+IUSE="handwriting +spell +xcb"
 
 DEPEND="
 	~dev-qt/qtcore-${PV}
 	~dev-qt/qtdeclarative-${PV}
 	~dev-qt/qtgui-${PV}
+	~dev-qt/qtsvg-${PV}
 	spell? ( app-text/hunspell )
 	xcb? ( x11-libs/libxcb )
```
Comment (Pesa) [inline dev-qt/qtvirtualkeyboard/qtvirtualkeyboard-5.7.1.ebuild]: does it link against libxcb? if so, add `:=`

**Comment by kensington** [comment]
Thanks, picked as eacb963efa9bf588b98ab2a8c156128818e3cf12...e5a6f897b984e718c7bea05675cb5d4d11751dea.

**Pull request closed without merging**